src/remarkable.ts

```diff
@@ -205,6 +215,7 @@ export default class Remarkable {
       }],
     });
     if (!body[0].Success || !body[0].BlobURLPut) {
+      console.warn('upload zip response: ', body[0]);
```

Comment:
do you want to include this information?
if so perhaps instead of console.warn the response can be appended to the error message